lib/src/over_react_redux/redux_multi_provider.dart

```diff
@@ -58,7 +58,7 @@ class _$ReduxMultiProviderProps extends UiProps {
   /// in a connected component is exactly the same as it would with a standard
   /// [ReduxProvider].
   @requiredProp
-  Map<Context, Store> storesByContext;
+  Map<Context, Store>? storesByContext;
```

Comment:
Based on the assertion that it is not null within `render()` below... it feels like this should be non-nullable required.

Reply:
Yeah, this is one of the components where we're updating nullability in your follow-up PR:  https://github.com/Workiva/over_react/pull/855/files#diff-d14bf2e659b0daff0280353a5d65b77a142da37d13c06061f7cf17d0e6f029c4R60